packages/flutter/lib/src/widgets/shortcuts.dart

```diff
@@ -868,6 +868,8 @@ class ShortcutManager with Diagnosticable, ChangeNotifier {
 /// This widget establishes a [ShortcutManager] to be used by its descendants
 /// when invoking an [Action] via a keyboard key combination that maps to an
 /// [Intent].
+/// 
+/// This is similar to but more powerful than the [CallbackShortcuts] widget.
```

Comment:
Maybe hint at why it's more powerful (separation of definition location for actions and intents, ability to override the action dispatcher, etc.).

Reply:
Added, let me know what you think!